[2025-03-19T22:22:24.816+0000] {local_task_job_runner.py:123} INFO - ::group::Pre task execution logs
[2025-03-19T22:22:24.862+0000] {taskinstance.py:2614} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: dag_with_taskflow_Api_v2.greet scheduled__2025-01-20T00:00:00+00:00 [queued]>
[2025-03-19T22:22:24.879+0000] {taskinstance.py:2614} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: dag_with_taskflow_Api_v2.greet scheduled__2025-01-20T00:00:00+00:00 [queued]>
[2025-03-19T22:22:24.879+0000] {taskinstance.py:2867} INFO - Starting attempt 1 of 6
[2025-03-19T22:22:24.921+0000] {taskinstance.py:2890} INFO - Executing <Task(_PythonDecoratedOperator): greet> on 2025-01-20 00:00:00+00:00
[2025-03-19T22:22:24.937+0000] {logging_mixin.py:190} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:70 DeprecationWarning: This process (pid=508) is multi-threaded, use of fork() may lead to deadlocks in the child.
[2025-03-19T22:22:24.941+0000] {standard_task_runner.py:72} INFO - Started process 514 to run task
[2025-03-19T22:22:24.948+0000] {standard_task_runner.py:104} INFO - Running: ['***', 'tasks', 'run', 'dag_with_taskflow_Api_v2', 'greet', 'scheduled__2025-01-20T00:00:00+00:00', '--job-id', '3154', '--raw', '--subdir', 'DAGS_FOLDER/dag_with_taskflow_api.py', '--cfg-path', '/tmp/tmpnf12zt7l']
[2025-03-19T22:22:24.952+0000] {standard_task_runner.py:105} INFO - Job 3154: Subtask greet
[2025-03-19T22:22:25.182+0000] {task_command.py:467} INFO - Running <TaskInstance: dag_with_taskflow_Api_v2.greet scheduled__2025-01-20T00:00:00+00:00 [running]> on host 63fd4b143e0a
[2025-03-19T22:22:25.473+0000] {taskinstance.py:3134} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='dag_with_taskflow_Api_v2' AIRFLOW_CTX_TASK_ID='greet' AIRFLOW_CTX_EXECUTION_DATE='2025-01-20T00:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2025-01-20T00:00:00+00:00'
[2025-03-19T22:22:25.477+0000] {taskinstance.py:732} INFO - ::endgroup::
[2025-03-19T22:22:25.480+0000] {logging_mixin.py:190} INFO - Hello JohnDoe, you are 19 years old.
[2025-03-19T22:22:25.481+0000] {python.py:240} INFO - Done. Returned value was: None
[2025-03-19T22:22:25.494+0000] {taskinstance.py:341} INFO - ::group::Post task execution logs
[2025-03-19T22:22:25.495+0000] {taskinstance.py:353} INFO - Marking task as SUCCESS. dag_id=dag_with_taskflow_Api_v2, task_id=greet, run_id=scheduled__2025-01-20T00:00:00+00:00, execution_date=20250120T000000, start_date=20250319T222224, end_date=20250319T222225
[2025-03-19T22:22:25.557+0000] {local_task_job_runner.py:266} INFO - Task exited with return code 0
[2025-03-19T22:22:25.617+0000] {taskinstance.py:3901} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2025-03-19T22:22:25.623+0000] {local_task_job_runner.py:245} INFO - ::endgroup::
